sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/ErrorCodeValue.java

```diff
@@ -8,7 +8,7 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 /**
- * Defines values for ErrorCodeValue.
+ * Defines values for TextAnalyticsErrorCode.
```

Comment:
Was this a manual change or a regeneration change? If the former this will be overwritten next time the implementation layer is regenerated.

Reply:
I think the refactor for name change made this change, will revert this.